docroot/sites/default/settings.php

```diff
@@ -160,7 +160,7 @@
   }
   // Use a dedicated branch to identify production environment.
   // This is useful when 'production' Lagoon environment is not provisioned yet.
-  elseif (getenv('LAGOON_GIT_BRANCH') == getenv('LAGOON_PRODUCTION_BRANCH')) {
+  elseif (getenv('LAGOON_GIT_BRANCH') == getenv('LAGOON_PRODUCTION_BRANCH', 'production')) {
```

Comment:
```suggestion
  elseif (getenv('LAGOON_GIT_BRANCH') !== FALSE && getenv('LAGOON_GIT_BRANCH') == getenv('LAGOON_PRODUCTION_BRANCH')) {
```

The second parameter does not set a default value it is a local_only flag: - Set to true to only return local environment variables (set by the operating system or putenv).

The suggested change ensures that a LAGOON_GIT_BRANCH has a value set before proceeding to check equality.